variants/arduino_nano_matter_mgm240sd22vna/arduino_nano_matter_mgm240sd22vna.overlay

```diff
@@ -0,0 +1,121 @@
+&flash0 {
+	partitions {
+		compatible = "fixed-partitions";
+		#address-cells = <1>;
+		#size-cells = <1>;
+
+		/* Arduino user sketch partition */
+		user_sketch: partition@C4000 {
+			reg = <0x000C4000 0x000B8000>;
+			label = "user";
+		};
+	};
+};
+
+/ {
+	zephyr,user {
+		digital-pin-gpios = <&gpioa 4 0>, /* D0 */
+				<&gpioa 5 0>, /* D1 */
+				<&gpioa 3 0>, /* D2 */
+				<&gpioc 6 0>, /* D3 */
+				<&gpioc 7 0>, /* D4 */
+				<&gpioc 8 0>, /* D5 */
+				<&gpioc 9 0>, /* D6 */
+				<&gpiod 2 0>, /* D7 */
+				<&gpiod 3 0>, /* D8 */
+				<&gpiod 4 0>, /* D9 */
+				<&gpiod 5 0>, /* D10 */
+				<&gpioa 9 0>, /* D11 */
+				<&gpioa 8 0>, /* D12 */
+				<&gpiob 4 0>, /* D13 */
+				<&gpiob 0 0>, /* A0 */
+				<&gpiob 2 0>, /* A1 */
+				<&gpiob 5 0>, /* A2 */
+				<&gpioc 0 0>, /* A3 */
+				<&gpioa 6 0>, /* A4 */
+				<&gpioa 7 0>, /* A5 */
+				<&gpiob 1 0>, /* A6 */
+				<&gpiob 3 0>, /* A7 */
+				<&gpioc 1 0>, /* 22 LED R */
+				<&gpioc 2 0>, /* 23 LED G */
+				<&gpioc 3 0>, /* 24 LED B */
+				<&gpioa 0 0>, /* 25 Button */
+				<&gpioc 4 0>, /* 26 Serial Tx */
+				<&gpioc 5 0>; /* 27 Serial Rx */
+
+		builtin-led-gpios = <&gpioc 1 GPIO_ACTIVE_LOW>, /* 22 LED R */
```

Comment:
@silabs-bozont could you apply the same changes we have in PR https://github.com/arduino/ArduinoCore-zephyr/pull/265 In that way LED_BUILTIN is green. Thanks.